public/i18n/de.json

```diff
@@ -191,6 +194,11 @@
         "LICENSE_LINK_AFTER": "gefunden werden"
     },
     "settings": {
-        "SETTINGS":"Einstellungen"
+        "SETTINGS":"Einstellungen",
+        "notifications": {
+            "NOTIFICATIONS" : "Benachrichtigungen",
+            "SHOW_NOTIFICATIONS" : "Desktopbenachrichtigungen anzeigen",
+            "SHOW_PREVIEW" : "Vorschau anzeigen"
```

Comment:
I used the strings provided by the Android App. I think it would be better to have consistent strings for the same setting(s) across the platforms.

But if you want i can change them, your decision ;)